stubs/mongodb.stub

```diff
@@ -0,0 +1,18 @@
+mongodb:
+    image: 'mongodb/mongodb-community-server:6.0-ubi8'
```

Comment:
I'll try to use the `mongodb/mongodb-atlas-local` image, so that atlas search is available. https://www.mongodb.com/docs/atlas/cli/current/atlas-cli-deploy-docker/